thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -438,6 +439,36 @@ public Response updateAnomalyFunction(@NotNull @QueryParam("id") Long id,
     return Response.ok(id).build();
   }
 
+  // Partially update anomaly function
+  @POST
```

Comment:
please use PUT as the method name instead of POST, then you do not need update/properties in your path. 
Also agree with yen, this should be put in AnomalyFunctionResource where the current 'UPDATE' endpoint located.